packages/react-dom/src/server/ReactPartialRenderer.js

```diff
@@ -80,9 +82,9 @@ if (__DEV__) {
     currentDebugStack = null;
     ReactDebugCurrentFrame.getCurrentStack = null;
   };
-  var getStackAddendum = function(): null | string {
+  getStackAddendum = function(): null | string {
     if (currentDebugStack === null) {
-      return null;
+      return '';
```

Comment:
I'm curious: Why the change from `null` => `''`?

(Is it to maintain the previous behavior where we had an inline function just returning an empty string?)

Reply:
Because we use `getStack()` directly as `%s`, which would show up as `null` right in the message when missing. I agree it's not great though and would be nice to fix the whole thing by adding a separate `warningWithStack` or something.

Reply:
Ah. Gotcha.

I asked b'c the Flow return type is also `null | string` but it's no big deal 😄